core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -155,6 +156,36 @@ struct SpaceAwareAccessor<AnonymousSpace, NestedAccessor> {
   friend struct SpaceAwareAccessor;
 };
 
+// Like atomic_accessor_relaxed proposed for ISO C++26 but with
+// defaulted memory scope - similar how desuls AtomicRef has a memory scope
+template <class ElementType, class MemoryScope = desul::MemoryScopeDevice>
+struct AtomicAccessorRelaxed {
+  using element_type = ElementType;
+  using reference =
+      desul::AtomicRef<ElementType, desul::MemoryOrderRelaxed, MemoryScope>;
+  using data_handle_type = ElementType*;
+  using offset_policy    = AtomicAccessorRelaxed;
+
+  static_assert(
+      std::is_same_v<std::remove_cv_t<element_type>, element_type>,
+      "AtomicAccessorRelaxed can only be used for non-const element types");
+  KOKKOS_FUNCTION
+  explicit operator default_accessor<element_type>() const {
+    return default_accessor<element_type>{};
+  }
```

Comment:
```suggestion
  static_assert(
      std::is_same_v<std::remove_cv_t<element_type>, element_type>,
      "AtomicAccessorRelaxed can only be used for non-const element types");

  KOKKOS_FUNCTION
  explicit operator default_accessor<element_type>() const {
    return default_accessor<element_type>{};
  }
```

Reply:
Yes. An empty line improves readability.

Reply:
Sure, I tend to agree with you on that one. But the way you selected more lines in your suggestion gets in the way.